playbook/colors.yml

```diff
@@ -0,0 +1,62 @@
+background_colors:
```

Comment:
this file might be better placed

Reply:
Would make sense to have this file more towards the root of the playbook folder. We do have a types folder where we store files with all the TS types for tokens/props. We could use the same folder or start a different one for rails? What do you think? I like the idea of what you're doing here!

Reply:
@nidaqg I would make a "labels" or "tokens" dir in the rails root and then add these yml files to it. We could also have a `sizes.yml` I see things like this all the time `%w[xxs xs sm md base lg xl]`

Reply:
I like the idea of having more yml files for types just like we do for TS. It'll definitely clean up a lot of our .rb files!

I'll defer to @jasperfurniss on where he thinks these files should live, we need to make sure it makes sense for file structure as a whole

Reply:
This has been removed from this PR as its out of scope